site-src/api-types/httproute.md

```diff
@@ -165,6 +165,27 @@ Service:
 Reference the [backendRef][backendRef] API documentation for additional details
 on `weight` and other fields.
 
+#### Timeouts (optional)
+
+Timeouts resources are API objects that provide request timeouts for use in `HTTPRoutes`. If unspecified, timeout behavior is implementation-specific.
+
+There are 2 kinds of timeouts that can be configured in an HTTPRouteRule:
+
+1. `request` is the timeout for the Gateway API implementation to send a response to a client HTTP request. Whether the gateway starts the timeout before or after the entire client request stream has been received, is implementation dependent. This timeout is intended to cover as close to the whole request-response transaction as possible, although an implementation MAY choose to start the timeout after the entire request stream has been received instead of immediately after the transaction is initiated by the client.
+
+2. `backendRequest` is a timeout for a single request from the gateway to a backend which may be used in conjunction with retry configuration (if supported by the implementation). This timeout covers the time from when the request first starts being sent from the gateway to when the full response has been received from the backend.
+
+Because the `request` timeout encompasses the `backendRequest` timeout, the value of `backendRequest` must be less or equal to the value of `request` timeout.
```

Comment:
```suggestion
Because the `request` timeout encompasses the `backendRequest` timeout, the value of `backendRequest` must not be greater than the value of `request` timeout.
```